### ISSUE TITLE
Vscode isn't recognizing EJS in the languages modes

### ISSUE THREAD

**Issue by minkir014** [Digitalbrainstem/ejs-grammar#43]
I tried to install your extension manually from github and compile it but that fails

## Steps to reproduce:
1. I clone your repository
2. I open it in vscode and I ran npm install
3. I moved the folder to the extension folder in the users folder

## Expected result:
It must work as this is the known way of compiling extensions. <strong>Note:</strong> Your extension appears in the list but it doesn't work.

**Comment by minkir014** [comment]
@Betanu701 I understood something. Even if I installed your extension via VSIX. It doesn't work. Vscode doesn't recognize ejs in the languages modes as before.



![too](https://user-images.githubusercontent.com/42807112/66542853-fe129180-eb33-11e9-9385-b83360cfe1ba.png)

**Comment by Betanu701** [comment]
Version 1.0+ does not show in the languages. Set the language to HTML. It will work.

**Comment by ckot** [comment]
I installed v1.1.3 into vscode via the marketplace.  I've setup *.ejs to use the html language.

The html syntax highlighting works fine, but not the EJS stuff.  Also, it lists linter warnings such as:
Special characters must be escaped: [ < ]. (spec-char-escape) ...

**Comment by minkir014** [comment]
I am facing the same issue.

**Comment by ckot** [comment]
In my case, it turned out not to be an issue with this extension, but rather my HTMLHint configuration, which was causing all of the `<%`'s. and` %>`'s to get underlined rather than highlighted, and listed lots of warnings.

after I added the following to my settings.json

```
"htmlhint.options": {
        "spec-char-escape": false,
        "doctype-first": false,
    },
```
 it's working great now.

**Comment by fkhan698** [comment]
Whenever I go to my settings.json I added that in. But for some reason I am getting a red squiggle line at the top. It looks like this 
<img width="345" alt="Screen Shot 2019-10-17 at 3 51 49 PM" src="https://user-images.githubusercontent.com/32605509/67046554-16674b00-f0f6-11e9-858d-e4cca661b45d.png">

**Comment by ckot** [comment]
I could be misunderstanding, but it looks like you merely append that snippet to the file. if so, that isn't the correct syntax, and it should look like:

```
{
     /* perhaps some stuff */
     "files.associations": {
          "*.ejs": "html"
     },
     "htmlhint.options":  {
         "spec-char-escape": false,
         "doctype-first": false
     },
    /* perhaps other stuff */
}
```

**Comment by fkhan698** [comment]
Hmm I pasted in your code but now it has a squiggle line on the "htmlhint.options"
<img width="309" alt="Screen Shot 2019-10-17 at 5 03 16 PM" src="https://user-images.githubusercontent.com/32605509/67051200-1e2bed00-f100-11e9-83fd-730d712db604.png">

**Comment by ExE-Boss** [comment]
Because you don’t have HTMLHint installed, so&nbsp;**VS&nbsp;Code** isn’t&nbsp;recognising the&nbsp;option.

**Comment by fkhan698** [comment]
Okay so I got my user settings all set up correctly. But the ejs still will not compile. It looks like this 
<img width="339" alt="Screen Shot 2019-10-18 at 12 23 53 PM" src="https://user-images.githubusercontent.com/32605509/67114776-4e2bcc80-f1a2-11e9-9443-164a5e560cd1.png">

**Comment by ckot** [comment]
I'm guessing in the bottom left corner of the window it will have linter errors/warnings.  In my case ,there where htmlhint warnings,  I'm guessing yours are something else.  if it lists error codes, google search them, and you'll be able to determine what extension is causing them, and then how to make changes in settings.json so that it doesn't report/display EJS tags as errors.

**Comment by fkhan698** [comment]
Yeah I have no errors either. I'm debating whether I should just start from scratch again

**Comment by fkhan698** [comment]
@minkir014 Did you ever end up figuring out what your issue was?

**Comment by minkir014** [comment]
It was working very well until the last update syntax work but the same thing is not for closing tags and others.

**Comment by fkhan698** [comment]
@minkir014 Did it ever look like this?
<img width="339" alt="Screen Shot 2019-10-18 at 12 23 53 PM" src="https://user-images.githubusercontent.com/32605509/67440740-c984e800-f5bf-11e9-9a16-a29e4fbc4399.png">

**Comment by Betanu701** [comment]
Closing tags and multiple lines should be working in version 1.2
